apps/codeforafrica/src/components/ProjectPageHeader/ProjectPageHeader.js

```diff
@@ -118,18 +111,28 @@ const ProjectPageHeader = React.forwardRef(
                 }}
               >
                 <ProjectTile {...tileProps} />
-                <ProjectTitle
-                  variant="h4"
-                  sx={{ mt: 2, typography: { md: "h2" } }}
-                >
-                  {title}
-                </ProjectTitle>
-                <RichTypography
-                  variant="body2"
-                  sx={{ mt: 2.5, typography: { md: "subheading" } }}
-                >
-                  {subtitle}
-                </RichTypography>
+                <RichText
+                  typographyProps={{
+                    variant: "h4",
+                  }}
+                  sx={(theme) => ({
+                    mt: 2,
+                    "& strong": {
+                      color: theme.palette.primary.main,
+                    },
+                  })}
```

Comment:
```suggestion
                    sx: {(theme) => ({
                      mt: 2,
                      "& strong": {
                        color: theme.palette.primary.main,
                      },
                    })}
                  }}
```
I think this `sx` if for the typography